simde/mips/msa/add_a.h

```diff
@@ -38,8 +38,6 @@ simde_v16i8
 simde_msa_add_a_b(simde_v16i8 a, simde_v16i8 b) {
   #if defined(SIMDE_MIPS_MSA_NATIVE)
     return __msa_add_a_b(a, b);
-  #elif defined(SIMDE_ARM_NEON_A32V7_NATIVE)
-    return vaddq_s8(vabsq_s8(a), vabsq_s8(b));
```

Comment:
If `SIMDE_MIPS_MSA_A32V7_NATIVE` is renamed to `SIMDE_ARM_NEON_A32V7_NATIVE` on line 233 of `simde/mips/msa/types.h` then I don't think these changes are needed as `simde_v16i8` will be a neon `int8x16_t` and not something that needs converting to a  `simde_v16i8_private` to access the `.neon` attribute.

Reply:
The reason why I changed it to r_.neon = xxx(a_.neon, b_neon):
The key point is not the a, and b. The problem is r_.
Return type of vaddq_s8 can't be casted to simde_v16i8 implicitly on MSVC build. 
I used `r_.neon = xxx(a_.neon, b_.neon);` because the definition of r_ can be reused.

Otherwise, we need to write:
```C
#elif defined(SIMDE_ARM_NEON_A32V7_NATIVE)
    simde_v16i8 r_;
    r.neon = vaddq_s8(vabsq_s8(a), vabsq_s8(b));
    return r_;
```

Or is there other better implementation?

Reply:
Using this function as an example, its return type is `simde_v16i8` ; as I said above, that is typedef to be a neon `int8x16_t` if my recommended change (`SIMDE_MIPS_MSA_A32V7_NATIVE` is changed to `SIMDE_ARM_NEON_A32V7_NATIVE`) is made. No cast problems.

https://github.com/simd-everywhere/simde/blob/f0f3d09547fefaca1134923ce0c0af1ca4a176b3/simde/mips/msa/types.h#L233-L234

Reply:
Reverted the files back